imports/api/NewGameUtils.js

```diff
@@ -70,37 +70,64 @@ export var NewGameUtils = {
 
 	// Helper function to ensure all players are selected
 	allPlayersSelected() {
-		return (Session.get("current_east") != Constants.DEFAULT_EAST && 
-	 			Session.get("current_south") != Constants.DEFAULT_SOUTH && 
-	 			Session.get("current_west") != Constants.DEFAULT_WEST && 
+		return (Session.get("current_east") != Constants.DEFAULT_EAST &&
+	 			Session.get("current_south") != Constants.DEFAULT_SOUTH &&
+	 			Session.get("current_west") != Constants.DEFAULT_WEST &&
 	 			Session.get("current_north") != Constants.DEFAULT_NORTH);
 	},
 
 	someoneBankrupt() {
-		return (Session.get("east_score") < 0 || 
+		return (Session.get("east_score") < 0 ||
 				Session.get("south_score") < 0 ||
 				Session.get("west_score") < 0 ||
 				Session.get("north_score") < 0);
 	},
 
 	someoneAboveMinimum(minimum) {
-		return (Session.get("east_score") > minimum ||
-				Session.get("south_score") > minimum || 
-				Session.get("west_score") > minimum ||
-				Session.get("north_score") > minimum);
+		return (Session.get("east_score") >= minimum ||
```

Comment:
This is a good fix.  This is not a comment, but forcing above 30,000 was apparently wrong.